pkg/apis/eventing/v1beta1/trigger_conversion.go

```diff
@@ -45,16 +45,16 @@ func (source *Trigger) ConvertTo(ctx context.Context, to apis.Convertible) error
 }
 
 // ConvertFrom implements apis.Convertible
-func (sink *Trigger) ConvertFrom(ctx context.Context, from apis.Convertible) error {
+func (sink *Trigger) ConvertFrom(_ context.Context, from apis.Convertible) error {
```

Comment:
why that ?

Reply:
IDE was complaining unused var and I listen to my IDE